ctl/sim_ctl/sim_config.sh

```diff
@@ -16,13 +16,12 @@ echo "###"
 # General
 ####################
 
-# create and clean-up sim_dir
+# create a new simulation run directory
 echo "simdir: "$sim_dir
-if [ -d "${sim_dir}" ]; then
+if [ -e "${sim_dir}" ]; then
   mv ${sim_dir} ${sim_dir}_bku$(date '+%Y%m%d%H%M%S')
-  #rm -f ${sim_dir:?}
-fi # -d sim_dir
-mkdir -pv $sim_dir
+fi
+mkdir -p $sim_dir
```

Comment:
Just curiosity, why you want to remove the `v`-option

Reply:
Not working in OpenBSD (didn't check POSIX), and the message is not needed for the user, I think.

Neither of these argument are essential, though.

Reply:
I checked newest version of POSIX: [no -v option form mkdir](https://pubs.opengroup.org/onlinepubs/9799919799/), so I propose not to use it.